style/infra_template.rubocop.yml

```diff
@@ -0,0 +1,87 @@
+## Infrastructure .rubocop.yml template
+
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  TargetRubyVersion: 3.1
+  SuggestExtensions: false
+  DisplayCopNames: true
+  Include:
+    - './Rakefile'
+    - './config.ru'
+    - '**/*.rb'
+    - '**/*.rake'
+  Exclude:
+    # autogenerated files (that we don't change) should go here
+    - 'bin/**/*'
+    - 'config/environments/*.rb'
+    - 'config/routes.rb'
+    - 'db/**/*'
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+    - 'vendor/**/*'
+
+Bundler/GemComment:
+  OnlyFor:
+    - restrictive_version_specifiers
+    - branch
+
+# Configuration parameters: AllowURI, URISchemes.
+Layout/LineLength:
+  Max: 120
+
+Metrics/BlockLength:
+  IgnoredMethods:
+    - SimpleForm.setup
+    - FactoryBot.define
+    - factory
+    - describe
+    - context
+
+# consider if refactoring will improve adherence to SOLID and/or improve readability
+#  Sometimes splitting the class up won't improve readablity, it just makes reader jump around in the code.
+Metrics/ClassLength:
+  Max: 150 # default 100
+
+# consider if refactoring will improve adherence to SOLID and/or improve readability
+#  Sometimes splitting the method up won't improve readablity, it just makes reader jump around in the code.
+#  Good method names help.
+Metrics/MethodLength:
+  Max: 15 # default 10
+
+RSpec/DescribeClass:
+  Exclude:
+    - 'spec/views/**/*_spec.rb' # In view tests, the view is named, not the class
+
+# do try to keep examples short (can you split out the setup from the test?)
+RSpec/ExampleLength:
+  Enabled: false
+
+# most tests should test a single thing
+RSpec/MultipleExpectations:
+  Max: 5 # default 1
+  Exclude:
+    - 'spec/features/**/*_spec.rb'
+
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Max: 4 # default: 3
+
+# "> 0" is not less readable than ".positive?", "== 0" than ".zero?", etc.
+Style/NumericPredicate:
+  Enabled: false
+
+Style/RegexpLiteral:
+  Enabled: false
```

Comment:
Can you say more about why we'd want this one turned off?

Reply:
maybe this is a discussion item.

My feeling is:  developers have different backgrounds.  Regex is a common toold, but the RegexpLiteral is not something I encountered before ruby.  For a non-ruby coder, it can make the code harder to read.   I'm not going to die on the hill, tho.

Reply:
same here - maybe this could be a dev planning discussion item.  I'm willing to bend.